packages/frontend-2/components/settings/workspaces/Regions.vue

```diff
@@ -1,30 +1,89 @@
 <template>
   <section>
     <div class="md:max-w-xl md:mx-auto pb-6 md:pb-0">
-      <SettingsSectionHeader title="Regions" text="Manage your data residency" />
       <SettingsSectionHeader
-        title="Default region"
-        text="The default region will be used to store project data for new projects"
-        subheading
+        title="Data residency"
+        text="Manage where your workspace data resides."
       />
-      <div class="pt-6">
-        <SettingsWorkspacesRegionsSelect
-          v-model="defaultRegion"
-          show-label
-          label="Default region"
-          :items="availableRegions || []"
-          :disabled="!availableRegions?.length || isLoading"
-        />
+      <CommonLoadingIcon
+        v-if="isQueryLoading || !workspace"
+        class="justify-self-center"
+      />
+      <template v-else-if="workspace.hasAccessToMultiRegion">
+        <template v-if="!workspace.defaultRegion">
+          <div class="p-4 bg-foundation border-outline-2 border text-body-xs">
```

Comment:
Just double checking, this one supposed to be `border-outline-2` and the other `border-outline-3`?